extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/local_inference/image_2_text.py

```diff
@@ -128,10 +130,12 @@ async def run_inference(self, prompt: Prompt, aiconfig: "AIConfigRuntime", optio
         completion_data = await self.deserialize(prompt, aiconfig, parameters)
         inputs = completion_data.pop("inputs")
 
-        model_name: str | None = aiconfig.get_model_name(prompt)
-        if isinstance(model_name, str) and model_name not in self.pipelines:
-            self.pipelines[model_name] = pipeline(task="image-to-text", model=model_name)
-        captioner = self.pipelines[model_name]
+        model_name = get_hf_model(aiconfig, prompt, self)
+        key = model_name if model_name is not None else "__default__"
```

Comment:
nit: double underscores are usually associated with dunder methods or special built-in values in Python. Would prefer a different value here To avoid any confusion

Reply:
can make a little enum class or something, like 

```
class Sentinel(Enum):
    DEFAUT = "DEFAULT"
```

Then check for that value wherever needed and supply the real default value in a specific use case